collections/card4l_byoa.yaml

```diff
@@ -1,4 +1,5 @@
 Name: Sentinel-1 GRD CARD4L Processing
+BYOA_ID: card4l
```

Comment:
It's admittedly harder to read but maybe we should call this `BYOAID` as we don't use any snake case for any of the keys. This would also be in line with the openEO Platform id we use which is `OpenEOPID`.

Reply:
OK